src/node/node_signature_verify.h

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the Apache 2.0 License.
+#pragma once
+
+#include "ccf/tx.h"
+#include "crypto/verifier.h"
+#include "node_signature_verify.h"
+
+namespace ccf
+{
+  static bool verify_node_signature(
```

Comment:
New common place to verify node signatures, with backwards compatibility with 1.x ledger baked it.